rules/docker-snmp-sv2.mk

```diff
@@ -1,7 +1,7 @@
 # docker image for snmp agent
 
 DOCKER_SNMP_SV2 = docker-snmp-sv2.gz
-$(DOCKER_SNMP_SV2)_PATH = $(DOCKERS_PATH)/docker-snmp-sv2
+$(DOCKER_SNMP_SV2)_PATH = $(DOCKERS_PATH)/docker-snmp-sv2$(ARCH_SUFFIX)
```

Comment:
>docker-snmp-sv2$(ARCH_SUFFIX) [](start = 42, length = 29)

Can we use the same image name?
Or normalized name as `docker-snmp-sv2-$SONIC_ARCH`

Reply:
Image name is not changed for dockers, only docker path has been changed.
I will check the template method to find way to resolve this

Reply:
During hiredis compilation I faced compilation issues because of locale difference in distributions.
Below is the change to fix it in the docker-snmp-sv2 

```
$ diff -Naur  dockers/docker-snmp-sv2/ dockers/docker-snmp-sv2-armhf/
diff -Naur dockers/docker-snmp-sv2/Dockerfile.j2 dockers/docker-snmp-sv2-armhf/Dockerfile.j2
--- dockers/docker-snmp-sv2/Dockerfile.j2       2019-06-06 15:39:20.663871975 +0530
+++ dockers/docker-snmp-sv2-armhf/Dockerfile.j2 2019-06-07 21:59:05.814780451 +0530
@@ -35,6 +35,17 @@
 {%- endfor %}
 {%- endif %}

+
+#####
+# Fix for hiredis compilation issues.
+RUN apt-get install -y locales
+RUN locale-gen "en_US.UTF-8"
+RUN dpkg-reconfigure --frontend noninteractive locales
+ENV LC_CTYPE=en_US.UTF-8
+RUN sed -i '/^#.* en_US.* /s/^#//' /etc/locale.gen
+RUN locale-gen
+#####
+
 # Install up-to-date version of pip
 RUN curl https://bootstrap.pypa.io/get-pip.py | python3.6
 RUN python3.6 -m pip install --no-cache-dir hiredis

```

Reply:
Reverted the changes of duplicate docker-snmp-sv2 and added the fix in template.